app/v1/connectors/ListBenefitConnector.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.connectors
+
+import config.AppConfig
+import javax.inject.{Inject, Singleton}
+import play.api.http.Status
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+import v1.models.request.listBenefits.ListBenefitsRequest
+import v1.models.response.listBenefits.ListBenefitsResponse
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class ListBenefitConnector @Inject()(val http: HttpClient,
```

Comment:
I think we could maybe just use the DeleteRetrieveConnector + service for this endpoint?